scripts/exregional_run_analysis.sh

```diff
@@ -535,6 +543,22 @@ done
 #
 #-----------------------------------------------------------------------
 # 
+if [ ${gsi_type} == "OBSERVER" ]; then
+  miter=0
+  if [ ${ens_type} == "MEAN" ]; then
+    lread_obs_save=.true.
+    lread_obs_skip=.false.
+  else
+    lread_obs_save=.false.
+    lread_obs_skip=.true.
+    ln -s ../../enseman/observer_gsi/obs_input.* .
+  fi
+else
+  miter=2
+  lread_obs_save=.false.
+  lread_obs_skip=.false.
+fi
+
```

Comment:
line 556-559 will hardwire miter, lread_obs_save and lread_obs_save for non-observer runs. I would suggest to remove line 556-559.

Reply:
Changes made accordingly. Thanks!